setup.py

```diff
@@ -84,13 +84,6 @@
     url = 'https://github.com/openfisca/openfisca-core',
     long_description=long_description,
     long_description_content_type='text/markdown',
-
-    data_files = [
-        (
-            'share/openfisca/openfisca-core',
-            ['CHANGELOG.md', 'README.md'],
```

Comment:
It will remove them from package build ?

Reply:
Not exactly, these are still packaged, but just not installed in the `share` folder anymore. I removed these lines as I was getting a deprecation warning.

Reply:
OK, thanks.